src/mo_pack/tests/test_decompress_rle.py

```diff
@@ -20,7 +20,7 @@ def test_no_mdi(self) -> None:
         src_buffer = np.arange(12, dtype=">f4").data
         result = decompress_rle(src_buffer, 3, 4)
         assert_array_equal(result, np.arange(12).reshape(3, 4))
-        assert result.dtype == np.dtype("=f4")
+        assert result.dtype == np.dtype("=f4")  # noqa: S101
```

Comment:
@ukmo-ccbunney Is this just a symptom of the tests being unnecessarily wrapped in a class?

These kinda look like `unittest` tests. Does making them simple `test_*` functions resolve this and remove the need for the `noqa` ?

Reply:
Turning these into simple `test_` functions still results in the same `S101 Use of assert detected` Ruff failure.

It's a slightly annoying rule for tests (although I understand the reasoning for production part of the code), and the general opinion seems to be to just add a per-file ignore, e.g. https://stackoverflow.com/questions/68428293/s101-use-of-assert-detected-for-python-tests.

Since there is only one instance of this happening in `mo-pack`, are you happy with the `noqa` in the file, or is it more "visible" as a per-file exclusion in `pyproject.toml` (as per geovista and others)?

Alternatively, I could raise an exception, but this seems overkill for tests?

Reply:
Nah, all good buddy. Let's roll with it.

Thanks for investigating further 💯